th2_data_services/data_source/lwdp/stub_builder.py

```diff
@@ -111,7 +111,6 @@ def template(self) -> dict:
         return {
             self.message_fields.DIRECTION: None,
             self.message_fields.SESSION_ID: BrokenMessage(),
-            self.message_fields.MESSAGE_TYPE: BrokenMessage(),
```

Comment:
Stub builder should return the message that looks like the message from LwDP. 
I didn't check, just make sure that it's so

Reply:
v3 structure doesn't have `MESSAGE_TYPE` in top level dictionary, that's why I removed it